src/components/ReviewCard.jsx

```diff
@@ -65,12 +65,15 @@ const ReviewCard = ({
           </p>
         )}
         <p>Total likes: {votes}</p>
-        <button onClick={upVotesHandler} disabled={hasClicked && !isError}>
+        <button
```

Comment:
I'm confused as to why you would vote on articles/reviews in this place, if I don't know the content of the article or review why would I up or down vote it?